README.md

```
@@ -33,6 +33,17 @@ bin/rails decidim:upgrade
 bin/rails db:migrate
 ```
 
+> ** MIGRATION TO v0.31:**
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)